R/as_rtf.R

```diff
@@ -0,0 +1,101 @@
+#' Print a summary table using gt
+#'
+#' Create RTF file created with \code{\link{as_table}} to summarize
+#' an object; currently only implemented for \code{\link{gsBinomialExact}}.
+#'
+#' @param x   Object to be display on rtf.
+#' @param ... Other parameters that may be specific the object.
+#'
+#' @return NULL.
+#'
+#' @seealso \code{vignette("binomialSPRTExample")}
+#'
+#' @details
+#' Currently only implemented for \code{\link{gsBinomialExact}} objects.
+#' Creates a table to summarize an object.
+#' For \code{\link{gsBinomialExact}}, this summarized operating characteristics
+#' across a range of effect sizes.
+#'
+#' @export
+#'
+#' @examples
```

Comment:
If you have the `@examples` section, it should have code examples and not left blank.